examples/js/index.js

```diff
@@ -1,3 +1,5 @@
+const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="top"]')
```

Comment:
So, here is I think the issue. `tooltipTriggerList` evaluates to an empty NodeList. I think we're running this code too early. If you want to run it AFTER page JS activity is complete, we can put it under the `(function() {` line. That defers execution until the page is ready. Can we do that? We may need to define it with global scope above, then set it once the page is ready.